### DISCUSSION TITLE
Hit sounds do not respect theme

### DISCUSSION THREAD

**Discussion by gllxflr** [ppy/osu#27655] [Q&A] [answered]
Laser 2024.312.1, Linux

No matter which theme I select (for example, Argon), I hear sounds from the Classic theme. This is really throwing me off.

2024.130.2 is unaffected

**Comment by peppy** [comment]
maybe you're just hearing the new hitsounds which were added in the last release.

**Reply by gllxflr** [reply]
is there a way to go back to the previous Argon ones? changing theme does not at all change hitsounds

**Reply by peppy** [reply, marked as answer]
The previous ones are now available via "Triangles".

In the future we may offer a way to change this.

**Reply by gllxflr** [reply]
Is there a way to use the sounds from "Triangles" with the Argon looks? Maybe a separate sound and visual theme selector could work for future

**Comment by nagi-desuuu** [comment]
Are your beatmap hitsounds turned on?

**Reply by gllxflr** [reply]
I can hear the beatmap-specific ones, just the default ones are different
